src/command_modules/azure-cli-cosmosdb/HISTORY.rst

```diff
@@ -1,5 +1,10 @@
 .. :changelog:
 
+unreleased
+===============
+
+* Minor fix allowing Creation of Collection with custom partition key
```

Comment:
I have made the requested changes, but I want to point out I followed the guidelines here:  https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log

Those probably should be updated to represent the new standards or modified to clarify what was intended. I'm still not 100% sure that I did what was intended, tried to read between the lines a bit.